<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QLineEdit
{
border-radius: 5px;
border: 2px solid #CCC;
}
QScrollArea
{
border-radius: 10px;
border: 2px solid #DDD;
}

QPushButton 
{
color: #111;
background-color: rgba(255, 255, 255, 0);
border-radius: 5px;
border: 2px solid #555;
padding-left:8px;
padding-right:8px;
padding-top:2px;
padding-bottom:2px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="main_widget" native="true">
     <property name="styleSheet">
      <string notr="true">#main_widget {
                         background: qlineargradient(
                            spread:pad,
                            x1:0, y1:1, x2:1, y2:1,
                            stop:0 #ff00ff,
                            stop:1 #00ff00
                        )
                     }
                     QWidget, QLabel, QPushButton {
                        color: #ffffff;
                     }</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="layout_list">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="but_file_dialog">
                 <property name="text">
                  <string>Load Subtitle</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../../../../../.designer/icons/file.png</normaloff>../../../../../.designer/icons/file.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="but_insert_text">
                 <property name="text">
                  <string>Insert Text</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../../../../../.designer/icons/written.png</normaloff>../../../../../.designer/icons/written.png</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLineEdit" name="line_uri">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgba(255, 255, 255, 0);
border-radius: 5px;
border: 0px solid #000;
</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>uri (path file)</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLineEdit" name="line_name_list">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">
QLineEdit	{
border-radius: 5px;
border: 2px solid #CCC;
	background-color: rgba(252, 252, 252, 120);
}</string>
             </property>
             <property name="placeholderText">
              <string>Введите название списка</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>2</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="layout_words"/>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QWidget" name="verticalWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="but_clear">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Очистить список</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="layout_save">
         <item>
          <widget class="QPushButton" name="but_cancel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
color: #111;
background-color: rgba(255, 255, 255, 0);
border-radius: 5px;
border: 2px solid #555
}</string>
           </property>
           <property name="text">
            <string>Cancle</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../../../.designer/icons/cancel.png</normaloff>../../../../../.designer/icons/cancel.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="but_save">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
color: #111;
background-color: rgba(255, 255, 255, 0);
border-radius: 5px;
border: 2px solid #555
}</string>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../../../.designer/icons/save.png</normaloff>../../../../../.designer/icons/save.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
